plugins/accelerated-peft/src/fms_acceleration_peft/framework_plugin_autogptq.py

```diff
@@ -103,12 +91,25 @@ def model_loader(self, model_name: str, **kwargs):
         )
         AutoModelForCausalLM.from_config = _from_config  # patch
 
+        if is_fsdp_enabled():
+            from .autogptq_utils import make_sure_no_tensor_in_meta_device
+            source = importlib.import_module("auto_gptq.modeling._utils")
+            original_obj = getattr(source, "make_sure_no_tensor_in_meta_device")
+            setattr(source, "make_sure_no_tensor_in_meta_device", make_sure_no_tensor_in_meta_device)
+            # reload and this should get the patched object
+            target_module = importlib.import_module("auto_gptq.modeling._base")
```

Comment:
can you define this function and put it in `autogptq_utils` first. If we move the model patcher to `framework` we will replace it 

```python
def _patch_target_module(
    to_patch: str,
    replace_with: Any,
    target_module: str = None,
):
    to_patch = to_patch.split('.')
    assert len(to_patch) > 1, "must have an object to patch"

    to_patch, obj_name_to_patch = to_patch[:-1], to_patch[-1]
    to_patch = ".".join(to_patch)
    source = importlib.import_module(to_patch)
    original_obj = getattr(source, obj_name_to_patch)
    setattr(source, obj_name_to_patch, replace_with)

    if target_module is not None:
        # reload and this should get the patched object
        target_module = importlib.import_module(target_module)
        importlib.reload(target_module)

        # replace it
        setattr(source, obj_name_to_patch, original_obj)

```

Reply:
and just call it 

```python
_patch_target_module(
   "auto_gptq.modeling._utils", make_sure_no_tensor_in_meta_device, "auto_gptq.modeling._base"
)
```